http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/auth/AllowlistAuthProvider.scala

```diff
@@ -159,4 +159,6 @@ class AllowlistAuthProvider(config: Config, saProvider: ServiceAccountProvider[I
       }
     }
   } yield filteredResources.toSet
+
+  override def isAdminUser(userInfo: UserInfo)(implicit ev: Ask[IO, TraceId]): IO[Boolean] = ???
```

Comment:
Can this be `F.pure(true)`? Probably shouldn't commit placeholder code..

Reply:
Hmm... I guess it's fine in Mock functions, but that seems to indicate that this code is never actually tested since `isAdminUser` is never hit on the mock 🤔